core/math/vector3.h

```diff
@@ -214,7 +214,7 @@ real_t Vector3::distance_squared_to(const Vector3 &p_b) const {
 
 real_t Vector3::angle_to(const Vector3 &p_b) const {
 
-	return Math::acos(this->dot(p_b) / Math::sqrt(this->length_squared() * p_b.length_squared()));
+	return Math::atan2(cross(p_b).length(), dot(p_b));
```

Comment:
I don't mind using either one, but here are two reasons on top of my mind: 1) it's faster 2) if we're going to down that road (where we perceive errors ~10^-8 as large, see the references "Mangled Angles"), we've got *a lot* of work to do, fixing the remainder of the code with precision issues in mind.

So long story short, the current PR sits well with the current state of the core/math.